Makefile

```diff
@@ -23,3 +23,7 @@ seed: db
 	cat ./priv/repo/seed.sql | docker exec -i $(db_container) psql -U postgres -d starknet_explorer_dev
 create-seed: db
 	docker exec -i $(db_container) pg_dump --column-inserts --data-only -d starknet_explorer_dev -U postgres > ./priv/repo/seed.sql
+
+juno_dev:
+	mkdir -p ./juno_files
+	docker-compose up juno_dev
```

Comment:
Either this should be `docker-compose up juno` or the service in the `docker-compose.yml` file should be renamed to `juno_dev`